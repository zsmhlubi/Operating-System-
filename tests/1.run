./witsshell tests/1.in
