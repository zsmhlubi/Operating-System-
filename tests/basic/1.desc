Input to check ls command without arguments
