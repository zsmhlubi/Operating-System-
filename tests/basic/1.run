./witsshell tests/basic/1.in
