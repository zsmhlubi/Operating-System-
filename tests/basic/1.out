tester
tests
tests-out
test-witsshell.sh
witsshell
witsshell.c
